R/as_draws_matrix.R

```diff
@@ -72,6 +72,22 @@ as_draws_matrix.draws_list <- function(x, ...) {
   as_draws_matrix(x, ...)
 }
 
+#' @rdname draws_matrix
+#' @export
+as_draws_matrix.draws_rvars <- function(x, ...) {
+  if (ndraws(x) == 0) {
+    return(empty_draws_matrix(variables(x)))
+  }
+  draws <- do.call(cbind, lapply(seq_along(x), function(i) {
```

Comment:
Sure, done. I had to make `enclos` an argument of `do_call` in order to support its usage in `rdo`. I'll open a separate comment for that in case you want a different solution.